src/python/pants/bin/daemon_pants_runner.py

```diff
@@ -102,6 +102,21 @@ def exit_code(self):
     return self._exit_code
 
 
+class _PantsProductPrecomputeFailed(Exception):
+  """
+  Represents a failure in the precompute step of v2 products for pants runs.
+  (e.g. The user called ./pants dependees and there was a broken link)
+
+  Will be raised by the DaemonPantsRunner at creation time.
+  """
+
+  def __init__(self, wrapped_exception: Exception) -> None:
```

Comment:
you wont be able to use this in 1.17.x which I am assuming you are aiming for?

Reply:
Indeed! Updated, thanks!